mod_datadog/CMakeLists.txt

```diff
@@ -77,14 +77,12 @@ if(HTTPD_DATADOG_ENABLE_COVERAGE)
         -fcoverage-mapping 
         -mllvm 
         -runtime-counter-relocation=true
-        -enable-name-compression=false
     )
     target_link_options(mod_datadog
       PRIVATE
         -fprofile-instr-generate 
         -mllvm 
         -runtime-counter-relocation=true
-        -enable-name-compression=false
```

Comment:
What was the usage of this flag ?

Reply:
As the name suggests, it serves as a flag to disable name compression.

Reply:
Where is it come from ? Because i saw nothing when i searched on internet